roles/solrcloud/molecule/default/converge.yml

```diff
@@ -5,7 +5,7 @@
     - running_on_server: false
   become: true
   pre_tasks:
-    - name: update cache
+    - name: Update cache
       ansible.builtin.apt:
         update_cache: true
         cache_valid_time: 600
```

Comment:
Since we no longer have any role dependencies, if we want other roles to run in testing, we need to add them below, I think.